modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/TestrayTask.ts

```diff
@@ -106,20 +106,30 @@ class TestrayTaskImpl extends Rest<TaskForm, TestrayTask, NestedObjectOptions> {
 	}
 
 	public async assignTo(task: TestrayTask, userIds: number[]) {
-		await this.update(task.id, {
+		const response = await this.update(task.id, {
 			dueStatus: TaskStatuses.IN_ANALYSIS,
 			name: task.name as string,
 		});
+
 		await this.assignUsers(task.id, userIds);
+
+		return response;
 	}
 
 	public async abandon(task: TestrayTask) {
-		return this.update(task.id, {
+		return await this.update(task.id, {
 			dueStatus: TaskStatuses.ABANDONED,
 			name: task.name,
 		});
 	}
 
+	public async complete(task: TestrayTask) {
+		return await this.update(task.id, {
+			dueStatus: TaskStatuses.COMPLETE,
+			name: task.name,
+		});
+	}
```

Comment:
No need to put async in this case

```suggestion
	public complete(task: TestrayTask) {
		return this.update(task.id, {
			dueStatus: TaskStatuses.COMPLETE,
			name: task.name,
		});
	}
```